src/pages/taeYeong/List/List.scss

```
@@ -0,0 +1,146 @@
+.list {
+  box-sizing: border-box;
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  p {
+    margin: 0;
+  }
+
+  .container {
+    width: 960px;
+    margin: 0 auto;
+    /* border: 1px solid black; */
+  }
+
+  .fas.fa-heart {
+    color: red;
+  }
+
+  #navbar {
+    height: 45px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #f4f4f2;
+    margin-bottom: 16px;
+  }
+
+  .navbar__logo {
+    font-family: 'Dancing Script', cursive;
+    font-size: 20px;
+    font-weight: bold;
+    margin-left: 20px;
+  }
+
+  .navbar__menu {
+    display: flex;
```

```suggestion
    .navbar__menu {
    display: flex;
        li {
        margin-right: 20px;
      }
         a {
    color: #b9b9b2;
    font-size: 13px;
    font-weight: bold;
  }
}
```
이것도 이렇게 네스팅하면 좋지 않을까요?